zod/src/__tests__/__snapshots__/zod.ts.snap

```diff
@@ -182,19 +166,7 @@ Object {
       "ref": undefined,
       "type": "invalid_type",
       "types": Object {
-        "invalid_type": Array [
-          "Expected number, received string",
-          "Expected undefined, received string",
-        ],
-        "invalid_union": "Invalid input",
-      },
-    },
-    "confirm": Object {
```

Comment:
I've opened an issue for that: https://github.com/colinhacks/zod/issues/586

Reply:
thanks @jorisre 👍